src/libmongoc/src/mongoc/mongoc-log.c

```diff
@@ -85,10 +103,8 @@ mongoc_log (mongoc_log_level_t log_level,
    bson_once (&once, &_mongoc_ensure_mutex_once);
 
    stop_logging = !gLogFunc;
```

Comment:
Should this check be combined into _mongoc_log_trace_is_enabled()?

Reply:
Unsure. I left the logic as close to the original as possible.